src/App.tsx

```diff
@@ -1,57 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Autocomplete } from './components/Autocomplete/Autocomplete';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [person, setPerson] = useState<Person>(peopleFromServer[0]);
```

Comment:
It is not necessary to set a specific person by default
<img width="620" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/54860641/b6bc0b41-3b3e-4233-9bd5-62e5f0cd551e">